src/main/kotlin/Graph.kt

```diff
@@ -0,0 +1,47 @@
+import Graph.Companion.STOCKHOLM
+import Graph.Companion.UPPSALA
+
+class Graph {
+    companion object {
+        const val STOCKHOLM = "ID_STOCKHOLM"
+        const val UPPSALA = "ID_UPPSALA"
+    }
+
+    val user: User? by lazy {
+        User()
+    }
+
+    fun findShortestPath(node1: Node, node2: Node): Int {
+        return 0
+    }
+
+    fun getNodeById(id: String): Node {
+
+        val firstName = user!!.firstName
+
+        return Node()
+    }
+}
+
+class Node
+
+class User {
+    val firstName = ""
+}
+
+fun main() {
+    // TODO: go through the rule set file and enable some rules
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tokgozmusa/detekt-danger/blob/0730a62ab6b603c21b1b0d6449937d0b3741a21f/src/main/kotlin/Graph.kt#L33"></span>This comment contains text that has been defined as forbidden in detekt.</td>
    </tr>
  </tbody>
</table>